contrib/containerized-pbench/README

```diff
@@ -0,0 +1,8 @@
+mkdir var_lib_pbench-agent
+mkdir fiotest
+vi example.sh && chmod 755 example.sh
+podman run --rm --network host --name example-with-bash -v ./example.sh:/example.sh:z -v ./var_lib_pbench-agent:/var/lib/pbench-agent:z -v ./fiotest:/fiotest:z quay.io/pbench/pbench-agent-all-centos-8:latest /example.sh
+podman run --rm --network host -it -v ./var_lib_pbench-agent:/var/lib/pbench-agent:z quay.io/pbench/pbench-agent-all-centos-8:latest bash
+  - Edit /opt/pbench-agent/config/pbench-agent.cfg to set target hostname
```

Comment:
s/target hostname/pbench_web_server/

And, if possible, it would be good to mention (somewhere) that it should include the port (if any) but exclude the protocol and URI.

Reply:
Perhaps this note should be in the `pbench-agent.cfg` file.

Reply:
Completely revamped how this all works.